apps/web/src/design-system/navigation/NavMenu.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+import { Tabs, Tab } from '@mantine/core';
+import useStyles from './NavMenu.styles';
+
+interface INavMenuProps {
+  menuItems: { icon: ReactNode; label?: string }[];
```

Comment:
```suggestion
  menuItems: { icon: ReactNode; label: string }[];
```

I think we can make the label mandatory, don't see a reason why it would be not mandatory. Wdyt?